Dockerfile

```
@@ -0,0 +1,22 @@
+# Use Python 3.11 slim image as base
+# TODO Use ultralytics for Jetson
+
+# Set working directory
+WORKDIR /app
```

Hadolint (2.14.0)

[error] Invalid instruction order. Dockerfile must begin with `FROM`, `ARG` or comment.

(DL3061)